conformal.py

```diff
@@ -44,15 +44,19 @@ def forward(self, *args, randomized=None, allow_zero_sets=None, **kwargs):
             randomized = self.randomized
         if allow_zero_sets == None:
             allow_zero_sets = self.allow_zero_sets
+
         logits = self.model_helper.get_logits(*args, **kwargs)
+        print("a", logits)
         logits = torch.from_numpy(np.array(logits))
+        print("b", logits)
+
         with torch.no_grad():
             logits_numpy = logits.detach().cpu().numpy()
             #TODO: is this axis=1 relates to image or logits? We should make parametric
-            scores = softmax(logits_numpy/self.T.item())#, axis=1)
-            I, ordered, cumsum = sort_sum(scores)
+            scores = softmax(logits_numpy/self.T.item(), axis=1)
+            I, ordered, cumsum = sort_sum_dataloader(scores)
```

Comment:
I remember this was not working with the fasttext example. But you havent run validate() yet, right?